### PR TITLE
assignment 3 documentation for google search API

### PR DESCRIPTION
**Pull request by blockchainguyy**

### CODE REVIEW

File: swagger.yaml [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+      produces:
+      - "application/json"
+      parameters:
+      - in: "body"
+        name: "body"
+        description: "Schema information for returned response"
+        required: true
+        schema:
+          $ref: "#/definitions/Search repositories"
+      - name: "q"
```
Comment (devanshutechracers) [inline swagger.yaml]: is q a `query` parameter? And if it is why is it in body as well. Do u REALLY get it what are `query` and `body` parameters?

File: swagger.yaml [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      url:
+        type: "string"
+        example: "https://api.github.com/users/dtrupenn"
+      received_events_url:
+        type: "string"
+        example: "https://api.github.com/users/dtrupenn/received_events"
+      type:
+        type: "string"
+        example: "User"
+
```
Comment (devanshutechracers) [inline swagger.yaml]: Only one new line is required at the end of file.

File: swagger.yaml [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+      name:
+        type: "string"
+        example: "Tetris"
+      full_name:
+        type: "string"
+        example: "dtrupenn/Tetris"
+      owner:
+        type: "object"
+        example:
+          $ref: "#/definitions/Owner"
```
Comment (devanshutechracers) [inline swagger.yaml]: is this `schema ref` or `example`?

Comment (devanshutechracers) [reply swagger.yaml]: did you do the self review before submission?

File: swagger.yaml [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+    properties:
+      total_count:
+        type: "integer"
+        example: 40
+      incomplete_results:
+        type: "boolean"
+        example: false
+      items:
+        type: "object"
+        example:
```
Comment (devanshutechracers) [inline swagger.yaml]: is this `schema ref` or `example`?

**Comment by rockon1985** [comment]
@ayushttiwari  _Google Search_ API ?? I thought it was Github's API. Check before you raise Pull request!

File: swagger.yaml [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+  description: "The Search API is optimized to help you find the specific item you're looking for (e.g., a specific user, a specific file in a repository, etc.). Think of it the way you think of performing a search on Google. It's designed to help you find the one result you're looking for (or maybe the few results you're looking for). Just like searching on Google, you sometimes want to see a few pages of search results so that you can find the item that best meets your needs. To satisfy that need, the GitHub Search API provides up to 1,000 results for each search."
+  version: "1.0.0"
+  title: "Search"
+host: "developer.github.com"
+basePath: "/v3"
+tags:
+- name: "Search repositories"
+  description: "Find repositories via various criteria. This method returns up to 100 results per page."
+schemes:
+- "http"
```
Comment (devanshutechracers) [inline swagger.yaml]: http !!!!!!!!!!!!!!!!!

File: swagger.yaml [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+        required: true
+        type: "string"
+        collectionFormat: "multi"
+      responses:
+        200:
+          description: "OK"
+          schema:
+            $ref: "#/definitions/Search"
+definitions:
+  Search repositories:
```
Comment (devanshutechracers) [inline swagger.yaml]: Is this a model?????

**Review by devanshutechracers — CHANGES_REQUESTED** [review]
- Add Google APIs in case of creating a pull request for GOOGLE SEARCH, not Github. 
- Read the API Documentation really carefully once again.

File: swagger.yaml [hunk 7]
```diff
@@ -0,0 +1,133 @@
...
+- name: "Search repositories"
+  description: "Find repositories via various criteria. This method returns up to 100 results per page."
+schemes:
+- "http"
+paths:
+  /search/repositories:
+    get:
+      tags:
+      - "Search repositories"
+      summary: "get all the film resources"
```
Comment (devanshutechracers) [inline swagger.yaml]: Copy + Paste should be done only when you can review all the things that you have copied.

**Review by devanshutechracers — CHANGES_REQUESTED** [review]
Incorrect Summary for APIs.

**Comment by blockchainguyy** [comment]
Sorry sir,
For making so-many silly mistakes. Will review all your comments properly &
will try to solve my errors on my own. I will make it a point to review my
assignments on my own before making a pull request. Sorry again.

Ayush Tiwari

**Comment by rockon1985** [comment]
Why is this PR having app.js and node_modules ?? It is supposed to be a swagger doc assignment PR. re-raise another pull request with just the swagger assignment!

**Pull request closed without merging**